### PR TITLE
Bugfix for set_header_str with python3

### PR DESCRIPTION
**Pull request by g-normand**
For set_header_str and set_footer_str, the code isn't working with python3

### CODE REVIEW

**Comment by g-normand** [comment]
Someone noticed the same error here : 
https://groups.google.com/forum/#!msg/python-excel/IGxA06wbElU/VX9-YzHsAAAJ

**Comment by jdufresne** [comment]
Can you add a test?

**Comment by g-normand** [comment]
@jdufresne  : I added some tests for worksheet properties

File: xlwt/Worksheet.py [hunk 1]
```diff
@@ -724,7 +724,7 @@ def get_horz_page_breaks(self):
     #################################################################
 
     def set_header_str(self, value):
-        if isinstance(value, str):
+        if not(PY3) and isinstance(value, str):
```
Comment (jdufresne) [inline xlwt/Worksheet.py]: Please write this as:

```py
if not PY3 and isinstance(value, str):
```

As `not` is not a function.

**Comment by g-normand** [comment]
@jdufresne  : I merged with the master branch to resolve the conflicts and I changed the code :) 

Everything should be ok now

**Comment by jdufresne** [comment]
Please rebase and squash the commits to just one. Please resolve Travis issues.

**Comment by g-normand** [comment]
@jdufresne : is it ok? :)

**Comment by jdufresne** [comment]
Thanks.

**Pull request merged**